src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="category">
+    <app-category-Edit
+      :category="targetCategory"
+      :access="access"
+      :error-message="errorMessage"
+      :done-message="doneMessage"
+      :disabled="disabled"
+      class="category-form"
+      @handle-submit="editCategory"
+      @update-value="updatedCategory"
+    />
+  </div>
+</template>
+
+<script>
+import { CategoryEdit } from '@Components/molecules';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  computed: {
+    categoryId() {
+      let { id } = this.$route.params;
+      id = parseInt(id, 10);
+      return id;
+    },
+    access() {
+      return this.$store.getters['auth/access'];
+    },
+    categories() {
+      return this.$store.state.categories.categoryList;
+    },
+    disabled() {
+      return this.$store.state.categories.disabled;
+    },
+    targetCategory() {
+      return this.$store.state.categories.targetCategory;
+    },
+    errorMessage() {
+      return this.$store.state.categories.errorMessage;
+    },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
+  },
+  created() {
+    this.$store.dispatch('categories/clearMessage');
+    this.$store.dispatch('categories/getCategory', parseInt(this.categoryId, 10));
+  },
+  methods: {
+    updatedCategory($event) {
```

Comment:
[nits]

`$` は Vue の組み込み関数として使いたい変数に付与される（$store など）ので、それと区別するために引数名などには使用しない方が良いかなと思いました。

```suggestion
    updatedCategory(event) {
```

Reply:
修正いたしました。